region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -179,6 +179,26 @@
       "notable": false,
       "requires": []
     },
+    {
+      "link": [1, 2],
+      "name": "Halfie Climb Room Stored Moonfall Grapple Clip",
```

Comment:
I dont think you need "Room". Should the name include "Door Lock Skip"? Same goes for the other changed bypass strat